lib/cocoapods/installer/xcode/pods_project_generator/app_host_installer.rb

```diff
@@ -79,7 +79,11 @@ def install!
             app_host_target.build_configurations.each do |configuration|
               configuration.build_settings['PRODUCT_NAME'] = app_target_label
               configuration.build_settings['PRODUCT_BUNDLE_IDENTIFIER'] = 'org.cocoapods.${PRODUCT_NAME:rfc1034identifier}'
-              configuration.build_settings['CODE_SIGN_IDENTITY'] = '' if platform == :osx
+              if platform == :osx
+                configuration.build_settings['CODE_SIGN_IDENTITY'] = ''
+              elsif platform == :ios
+                configuration.build_settings['CODE_SIGN_IDENTITY'] = 'iPhone Developer'
```

Comment:
the default now is `nil` from Xcode 12 build settings. I decided to maintain what we used to do.